### DISCUSSION TITLE
any way to facilitate translating the interface to another language the application?

### DISCUSSION THREAD

**Discussion by cochayullo** [DefectDojo/django-DefectDojo#4897] [Ideas]
Hello,
Is there or could there be any form or tool to be able to translate the interface into any other language, as an example: Spanish, Portuguese, etc?

regards!

**Comment by damiencarol** [comment]
Sadly we didn't implemented a I8n system in the UI. IT could be easy to do it as we rely on Django framework for now (2021)
